tests/test_besttrack.py

```diff
@@ -32,7 +32,24 @@ def test_from_fort22(self):
             with open(reference_filename) as reference_file:
                 assert test_file.read() == reference_file.read()
 
-    def test_external_bt(self):
-        input_external = INPUT_DIRECTORY / "test_besttrack" / "external_track.trk"
-        bt = BestTrackForcing("external_track", external_track=input_external)
-        bt.plot_track(show=True)
+    def test_from_atcf(self):
+        input_filename = INPUT_DIRECTORY / "test_besttrack" / "florence2018_atcf.trk"
+        output_filename = OUTPUT_DIRECTORY / "test_besttrack" / "florence2018_fort.22"
+        reference_filename = REFERENCE_DIRECTORY / "test_besttrack" / "florence2018_fort.22"
+
+        if not output_filename.parent.exists():
+            output_filename.parent.mkdir(parents=True, exist_ok=True)
+
+        best_track = BestTrackForcing.from_atcf_file(
+                atcf=input_filename,
+                nws=8,
+        )
+
+        assert best_track.storm_id == 'BT02008'
+        assert best_track.name == 'WRT00001'
+
+        best_track.write(output_filename, overwrite=True)
```

Comment:
You might want to call the `plot_track` method here. In generally these plots can be suppressed by calling `pytest` like 
```
MPLBACKEND=Agg pytest --maxfail=1
```

Reply:
You can use MPLBACKEND=Agg but I prefer to use the the unittests.patch decorator: `@patch('matplotlib.pyplot.show')`

Reply:
added in 2ed9c3e3fb91739013048f5fff19892efeee6fc7